### DISCUSSION TITLE
Extend 5 minute access to supporter features

### DISCUSSION THREAD

**Discussion by ghost** [sandboxie-plus/Sandboxie#3765] [Various ideas and suggestions]
I want to test "RestrictDevices=y" without having a supporter certificate to see if I need that feature, but the 5 minute limit is just not enough. It simply isn't possible to test some restrictions in such a small time frame. I doubt a longer time frame, such as 30 minutes, would dissuade anyone from becoming a supporter if additional features provided to supporters are deemed necessary after testing them.

**Comment by bastik-1001** [comment]
5 minutes is a tight fit, for testing some program features of Sandboxie-Plus and of the program to be run it. It might be too short, so it may hinder someone liking the feature enough to get a certificate.

**Comment by ghost** [comment]
I agree, especially if one is interested in "SysCallLockDown=y". It requires a lot of testing to find compatible settings for that feature.
